plugins/net_plugin/net_plugin.cpp

```diff
@@ -1663,8 +1669,8 @@ namespace eosio {
                ("r", sync_last_requested_num)("e", sync_next_expected_num)("k", sync_known_lib_num)("s", sync_req_span) );
 
       if( fork_head_block_num < sync_last_requested_num && sync_source && sync_source->current() ) {
-         fc_ilog( logger, "ignoring request, head is ${h} last req = ${r} source is ${p}",
-                  ("h", fork_head_block_num)( "r", sync_last_requested_num )( "p", sync_source->peer_name() ) );
+         fc_ilog( logger, "ignoring request, head is ${h} last req = ${r}",
```

Comment:
was there an intention to call peer_* macro or use connection_id instead?

Reply:
Could add in `sync_source->connection_id` wouldn't hurt